elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -2,31 +2,37 @@
 @import "_shared-assets.scss";
 @import "../../pfe-layouts/src/layouts/_grid";
 
+
+#main-menu {
+    background: transparent !important; // Remove this.
+}
+
 // IE11 styles
 @include browser-query(ie11) {
 
-    pfe-navigation {
-        max-height: 80px;
-    }
     pfe-navigation-item {
         [slot="trigger"] {
-            font-size: 12px;
+            font-size: 12px; 
+            margin: 0;
 
             a {
                 color: #{pfe-color(surface--darkest--text)};
             }
         }
+        // Accordion trigger styles when expanded.
```

Comment:
```suggestion
        // Accordion trigger styles when expanded.
        > * {
            margin: 0 10px 0 0;
        }
```

Reply:
I'm hoping these dont impact regular styles when javascript is enabled. If they do, forget about it :)